apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/RDASink.java

```diff
@@ -0,0 +1,48 @@
+package gov.cms.bfd.pipeline.rda.grpc;
+
+import java.util.Collections;
+
+/**
+ * Interface for objects that process incoming objects. At least one of the methods must be
+ * implemented. All implementations are AutoCloseable since they will generally hold a database
+ * connection.
+ *
+ * @param <T> the type of objects processed
+ */
+public interface RDASink<T> extends AutoCloseable {
```

Comment:
I would probably also add a constraint here and/or in the methods that's something like "all implementations of this interface SHALL be idempotent." (And explain what that means, as it'll be a new term for many folks.)